standalone/ansible/roles/simple-server/templates/.env.j2

```diff
@@ -10,9 +10,9 @@ SENDGRID_USERNAME={{ secrets.server.config.sendgrid_username }}
 SENDGRID_PASSWORD={{ secrets.server.config.sendgrid_password }}
 
 SENTRY_DSN={{ secrets.server.config.sentry_dsn }}
-SENTRY_CURRENT_ENV={{ deploy_env }}
+SENTRY_CURRENT_ENV={{ app_env }}
 
-SIMPLE_SERVER_ENV={{ deploy_env }}
+SIMPLE_SERVER_ENV={{ app_env }}
```

Comment:
What's the difference between app env and deploy env? `app_env=production` and `deploy_env=ethiopia-production`?

Reply:
The `deploy_env` is an identifier for the deployment (country + stage), `app_env` is for the rails app (`demo`, `sandbox` etc.). Ref: https://github.com/simpledotorg/deployment/pull/217#discussion_r438003054

Reply:
Love it. I knew I thought of this before. 😂  Nice job closing that loop.